Update deploy-a-hugo-site.md (#5089)
cloudflare/cloudflare-docs: content/pages/framework-guides/deploy-a-hugo-site.md

{{</Aside>}}

> Set the environment variable `HUGO_VERSION` in your Worker > **Settings** to use a [specific version of Hugo](https://github.com/gohugoio/hugo/releases).

Reviewer: ```suggestion
Set the environment variable `HUGO_VERSION` in your Worker > **Settings** > **Variables** to use a [specific version of Hugo](https://github.com/gohugoio/hugo/releases).
```

For example, `HUGO_VERSION`: `0.101.0`
  
{{<Aside type="note"}} 
         

> Since Hugo is constantly receiving new updates, it is strongly recommended to use a recent version of Hugo to avoid build errors.

Reviewer: ```suggestion
With Hugo receiving regular updates, it is recommended to use a recent version of Hugo to avoid build errors.
```
